src/Item.js

```diff
@@ -71,6 +109,27 @@ function Item({ userToken, item }) {
     setChecked(!checked);
   };
 
+  // [delete-item] 4a. If yes, then remove from firestore database
+  const handleDelete = () => {
+    setOpenModal(true);
+    db.collection(userToken)
+      .doc(id)
+      .delete()
+      .then(() => {
+        console.log('it worked!');
+      });
+  };
+
+  // [delete-item] 4b. If no, then the modal closes
+  const handleCloseModal = () => {
+    setOpenModal(false);
+  };
+
+  // [delete-item] 3. Create a function that is passed to onClick which will render a mondal which asks the user 'You sure about dat?'
```

Comment:
😂

Reply:
I forgot that comment was still in there 😂.